Lib/distutils/sysconfig.py

```diff
@@ -205,7 +205,7 @@ def customize_compiler(compiler):
         if 'LDFLAGS' in os.environ:
             ldshared = ldshared + ' ' + os.environ['LDFLAGS']
         if 'CFLAGS' in os.environ:
-            cflags = opt + ' ' + os.environ['CFLAGS']
+            cflags = cflags + ' ' + os.environ['CFLAGS']
```

Comment:
If my concern about `opt` turns out to not be valid and the decision is to move forward with the original proposed changed, the `opt` variable, and it's acquired value from`get_config_vars([...], 'OPT', [...])`, can be removed since it would be unused code.

Reply:
Right. Fixed.